package.json

```
@@ -206,27 +207,24 @@
     "cover-unit": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/unit/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/unit.info')\"",
     "merge-coverage": "lcov-result-merger \"coverage/*.info\" \"coverage/lcov.info\"",
     "----------------------------------------- local test dbs ------------------------------------------": "",
-    "start-mariadb": "bash dev/mariadb/10.3/start.sh",
-    "start-mysql": "bash dev/mysql/5.7/start.sh",
-    "start-mysql-8": "bash dev/mysql/8.0/start.sh",
-    "start-postgres": "bash dev/postgres/10/start.sh",
+    "start-mariadb-oldest": "bash dev/mariadb/oldest/start.sh",
+    "start-mariadb-latest": "bash dev/mariadb/latest/start.sh",
+    "start-mysql-oldest": "bash dev/mysql/oldest/start.sh",
+    "start-mysql-latest": "bash dev/mysql/latest/start.sh",
+    "start-postgres-oldest": "bash dev/postgres/oldest/start.sh",
+    "start-postgres-latest": "bash dev/postgres/latest/start.sh",
     "start-mssql-oldest": "bash dev/mssql/oldest/start.sh",
     "start-mssql-latest": "bash dev/mssql/latest/start.sh",
-    "start-db2": "bash dev/db2/11.5/start.sh",
-    "stop-mariadb": "bash dev/mariadb/10.3/stop.sh",
-    "stop-mysql": "bash dev/mysql/5.7/stop.sh",
-    "stop-mysql-8": "bash dev/mysql/8.0/stop.sh",
-    "stop-postgres": "bash dev/postgres/10/stop.sh",
-    "stop-mssql-oldest": "bash dev/mssql/oldest/stop.sh",
-    "stop-mssql-latest": "bash dev/mssql/latest/stop.sh",
-    "stop-db2": "bash dev/db2/11.5/stop.sh",
-    "restart-mariadb": "yarn start-mariadb",
-    "restart-mysql": "yarn start-mysql",
-    "restart-postgres": "yarn start-postgres",
-    "restart-mssql-oldest": "yarn start-mssql-oldest",
-    "restart-mssql-latest": "yarn start-mssql-latest",
-    "restart-db2": "yarn start-db2",
-    "----------------------------------------- local tests ---------------------------------------------": "",
+    "start-db2-oldest": "bash dev/db2/oldest/start.sh",
+    "start-oldest": "concurrently \"npm:start-*(!db2)-oldest\" && npm run start-db2-oldest",
```

```suggestion
    "start-oldest": "concurrently \"npm:start-*-oldest\"",
```
Any reason not to do this?

---

DB2 requires root (with the current implementation at least), starting/stopping it in parallelp leads to the password prompt being lost in the noise